gammapy/obs/tests/test_observation.py

```diff
@@ -186,3 +194,90 @@ def test_select_sky_regions():
                      lon=lon_cen, lat=lat_cen,
                      radius=radius, border=border)
     common_sky_region_select_test_routines(obs_table, selection)
+
+
+@remote_data
+def test_ObservationGroups(tmpdir):
+
+    # test create obs groups
+    alt = Angle([0, 30, 60, 90], 'degree')
+    az = Angle([-90, 90, 270], 'degree')
+    ntels = np.array([3, 4])
+    list_obs_group_axis = [ObservationGroupAxis('ALT', alt, 'bin_edges'),
+                           ObservationGroupAxis('AZ', az, 'bin_edges'),
+                           ObservationGroupAxis('N_TELS', ntels, 'bin_values')]
+    obs_group = ObservationGroups(list_obs_group_axis)
+    assert ((0 <= obs_group.obs_groups_table['GROUP_ID']) &
```

Comment:
It will be equivalent in the case of `AND`, but not in the case of `OR`. And I don't like splitting it because logically they belong together: it's an assert on the range of validity of one parameter. Splitting in 2 asserts it looks like it's 2 different tests, so it remains unchanged.